de_web_plugin.cpp

```diff
@@ -8101,57 +8128,15 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
             return;
         }
 
-        // filter for relevant clusters
-        if (event.profileId() == HA_PROFILE_ID || event.profileId() == ZLL_PROFILE_ID)
+        if (event.clusterId() == VENDOR_CLUSTER_ID && !(i->modelId() == QLatin1String("de_spect") && existDevicesWithVendorCodeForMacPrefix(event.node()->address(), VENDOR_DDEL)))
```

Comment:
The `"de_spect"` check can be removed. This was only a internal test device.

Reply:
You got it, done.